### ISSUE TITLE
Error Running the example, not all SDP Fields Provided

### ISSUE THREAD

**Issue by Meeshbhoombah** [kyren/webrtc-unreliable#16]
When running the example, I'm met with an error as a response (as seen in the debug console).
<img width="1440" alt="Screen Shot 2021-03-27 at 7 32 52 PM" src="https://user-images.githubusercontent.com/8546360/112740624-3b8afa80-8f33-11eb-8fc8-60bb37adf21b.png">

```
error: error streaming the incoming SDP descriptor: not all SDP fields provided
```

**Comment by jaburns** [comment]
This happens to me on head of master (43527e9) as well, if I go back to the commit tagged v0.5.0 I no longer get this error.

**Comment by kyren** [comment]
This should be fixed by 49ed336

**Comment by kyren** [comment]
Actually no hold on, I may have tested this badly, still working on it..

**Comment by kyren** [comment]
Okay *now* it's fixed, by 70eb324b0f8ebc1de35bec6406003d5d6787f59f

I don't fully understand all the details of *why* this last change was required, the best I can figure was I was setting js event handlers after the event had already happened, but I'm still not totally sure.  If anybody who is an actual javascript programmer and has actual knowledge of this wants to chime in that would be very welcome.

**Comment by Meeshbhoombah** [comment]
Thanks for the fix!
